### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by DevKaranJ**
This pull request implements the fourth project for my portfolio website. The project focuses on ensuring the website displays correctly on desktop screens using media queries.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -413,3 +426,220 @@ li::marker {
...
+
+  .submitButton {
+    position: relative;
+    left: 6rem;
+  }
+
+  .bottomLine {
+    display: none;
+  }
+}
```
Comment (AbayomiOlaoye) [inline style.css:645]: - Hey Haran! :wink: That was  so close :clap: :clap: 
- [ ] You have made commendable efforts on this but I think you could still improve on what we have now. The page is not correctly responsive in such a way that every user on every device is able to interact and navigate your page impressively. Check out this [video](https://watch.screencastify.com/v/JDdtbcx4XSPQ7faAPnf3) and see what I'm talking about.
  - [ ] In the video, you will see that your elements are not really responsive in terms of `font-sizes`, `color`, outline on project cards, etc. with regards to screen sizes as we have in the Figma design:
  - Desktop View:  
image
  - Mobile View:
image

  - [ ] Your project cards are not responsively centralized
  - [ ] We have an overflow on the y-axis
  - [ ] Your list of `language`, `skills`, and `frameworks` seems not to be responsive

- Kindly check the Figma design again and see how to improve what you have coded

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @DevKaranJ :wave: , 

Good job with this milestone so far! What a brilliant attempt!!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark:
- Good attempt :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -186,6 +197,7 @@ <h2>LET'S CONNECT</h2>
             </section>
         </section>
         <!--the contact section-->
+        <footer class="footer">
```
Comment (tufoinnkuo10) [inline index.html:200]: - [ ] please the headline, the about and contact section should be centralized and responsive as well. check here for the orignal design to be followed. nice work
yours
Screenshot (1245)_LI

Screenshot (1247)_LI

Screenshot (1248)_LI

File: index.html [hunk 3]
```diff
@@ -124,8 +134,9 @@ <h2>Tonic</h2>
                     </div>
                     <!--fourth card-->
                     <div class="card">
-                        <div class="workHeadline">
-                            <img src="./resource/project4.png" alt="work">
+                        <div class="workHeadline2">
```
Comment (tufoinnkuo10) [inline index.html:137]: - [ ] like the previous reviewer noted, your cards and all sections of your desktop version are not centralize. Kindly edit and remove the max-width you gave some sections, the margin left you gave them, so they can be centralized. The card content should also be worked in terms of padding, margins, spacing and aligning the content. Please try as much as possible to follow the [figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&mode=design&t=DE14pRV1XoxpS1y8-0)design. 
- [optional] kindly add the hover and pressed effects on your buttons as in the original version. 
yours
Screenshot (1246)_LI
figma
Screenshot (1249)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @DevKaranJ 
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 4]
```diff
@@ -43,7 +51,8 @@ <h1>I’m Karan.
             <!--first card-->
             <div class="card">
             <div class="workHeadline">
-                <img src="./resource/work.png" alt="work">
+                <img src="./resource/work.png" alt="work" class="mobile">
+                <img src="./resource/desktoppro1.png" alt="work" class="desktop">
                 <div class="workHeadlineText">
                     <h2>Tonic</h2>
                     <p class="heading">
```
Comment (Qoosim) [inline index.html:58]: - [ ] As noted by the previous reviewers, the project card is supposed to be at the center of your page and the width of the project info inside the card need to be increased too. Kindly check the screenshots below and walk your design alongside the figma design.


### Your Design

your-project

### Figma Design

temp-project


- [ ] As noted by the previous reviewers, your about myself section does not match the figma design. Kindly check the screenshots below and walk your design alongside the figma design.

### Your Design

your-about-me

### Figma Design

temp-about-me

- [ ]  While working on the desktop version of your portfolio, kindly ensure that your mobile version styling is not affected. I wish you the best.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @DevKaranJ, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kamba56 — APPROVED** [review]
Hi @DevKaranJ, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] Great job using media queries to make page responsive!👏👏👏
- [x] Great job using grid and flexbox to layout contents!👏👏👏
- [x] !👏👏👏

## Optional suggestions

- [OPTIONAL] Your portfolio looks great up to screen size with a width of 1024px. Your portfolio starts breaking at screen sizes above 1100px. Kindly look into that so that your project can be responsive on all screensizes.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**